code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -78,31 +74,24 @@
 	return
 
 /obj/structure/chair/proc/handle_rotation(direction)
+	handle_layer()
 	if(has_buckled_mobs())
 		for(var/m in buckled_mobs)
 			var/mob/living/buckled_mob = m
-			buckled_mob.buckled = null //Temporary, so Move() succeeds.
-			if(!direction || !buckled_mob.Move(get_step(src, direction), direction))
```

Comment:
Line 33 was the only time this was called. Take a good look, then consider that chairs worked fine without the second part of the `||`.